kubeadm/scripts/InstallNssm.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = "Stop"
+$global:NssmInstallDirectory = "$env:ProgramFiles\nssm"
+$arch = "win32"
+if ([Environment]::Is64BitOperatingSystem) {
+    $arch = "win64"
+}
+
+mkdir -Force $global:NssmInstallDirectory
+curl.exe -Lo nssm.zip https://nssm.cc/release/nssm-2.24.zip
+tar C $global:NssmInstallDirectory -xvf .\nssm.zip --strip-components 2 */$arch/*.exe
```

Comment:
We had to mirror the nssm bits due to frequent flakes observed when pulling form https://nssm.cc

For Azure we re-hosted it at https://k8stestinfrabinaries.blob.core.windows.net/nssm-mirror/nssm-2.24.zip